nix/package.nix

```diff
@@ -9,14 +9,18 @@ writeShellScriptBin "agenix" ''
 
   function die() { echo "[1;31merror:[m $*" >&2; exit 1; }
   function show_help() {
-    echo 'Usage: agenix [COMMAND]'
+    echo 'Usage: agenix <OPTIONS> [COMMAND]'
     echo "Edit, generate or rekey secrets for agenix."
     echo "Add help or --help to a subcommand to view a command specific help."
     echo ""
     echo 'COMMANDS:'
     echo '  rekey                   Re-encrypts secrets for hosts that require them.'
     echo '  edit                    Create/edit age secret files with $EDITOR and your master identity'
     echo '  generate                Automatically generates secrets that have generators'
+    echo ""
+    echo 'OPTIONS:'
+    echo '  --show-trace            Show the trace agenix.  This must be provided before the subcommand'
```

Comment:
Im not entirely sure whet `Show the trace agenix` means exactly 😅 Can we reword this?

Reply:
Derp!  I don't know what happened here.   This definitely deserves rewording!